app/ui-sidenav/client/sidebarItem.js

```diff
@@ -201,6 +201,9 @@ Template.sidebarItem.events({
 });
 
 Template.sidebarItemIcon.helpers({
+	uid() {
+		return this.rid.replace(this.u._id, '');
```

Comment:
can this replace be done only if direct message?

Reply:
hmmm I dont think we gonna have the "presence status" if its not a direct message, that why I didnt insert a test here